docs/tools/tools_intro.md

```diff
@@ -23,7 +23,7 @@ Mbed OS 5 can be built with various toolchains. The currently supported versions
 
 - [Arm compiler 5.06 update 6](https://developer.arm.com/products/software-development-tools/compilers/arm-compiler-5/downloads).
 - [GNU Arm Embedded version 6](https://developer.arm.com/open-source/gnu-toolchain/gnu-rm/downloads).
```

Comment:
Please specify which version of GNU Arm toolchain as well, there's multiple ones and the page seems to propose Version 8-2018-q4-major. Which is or is not aligned with anything else?

Reply:
@AnotherButler If it helps, we'll be specifically using `GCC Arm Compiler 6-2017-q2-update` (https://github.com/ARMmbed/mbed-os/pull/9844/files#diff-354f30a63fb0907d4ad57269548329e3R30)

https://developer.arm.com/open-source/gnu-toolchain/gnu-rm/downloads

The GNU Arm Embedded is what the Ubuntu package calls the compiler. I don't know why they name them differently.

Reply:
@ARMmbed/mbed-os-test-team Can confirm, but I suspect this is the same version in Jenkins CI as well.

Reply:
GCC version in CI is 6-2017-q1-update